syntax = "proto3";

import "corpc_option.proto";

option cc_generic_services = true;

message FooRequest {
    string msg1 = 1;
    string msg2 = 2;
}

message FooResponse {
    string msg = 1;
}

service HelloWorldService {
    option (corpc.global_service_id) = 1;

    rpc foo(FooRequest) returns(FooResponse);
}
